src/plugins/home/public/application/components/solutions_section/solution_panel.tsx

```diff
@@ -6,82 +6,50 @@
  * Side Public License, v 1.
  */
 
+import { snakeCase } from 'lodash';
 import React, { FC, MouseEvent } from 'react';
-import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
+import { EuiCard, EuiFlexItem, EuiToken } from '@elastic/eui';
 import { METRIC_TYPE } from '@kbn/analytics';
-import { FeatureCatalogueEntry, FeatureCatalogueSolution } from '../../../';
+import { FeatureCatalogueSolution } from '../../../';
 import { createAppNavigationHandler } from '../app_navigation_handler';
-import { SolutionTitle } from './solution_title';
 import { getServices } from '../../kibana_services';
 
-const getDescriptionText = (description: string): JSX.Element => (
-  <EuiText size="s" key={`${description}`}>
-    <p>{description}</p>
-  </EuiText>
-);
-
-const addSpacersBetweenElementsReducer = (
-  acc: JSX.Element[],
-  element: JSX.Element,
-  index: number,
-  elements: JSX.Element[]
-) => {
-  acc.push(element);
-  if (index < elements.length - 1) {
-    acc.push(<EuiSpacer key={`homeSolutionsPanel__UseCaseSpacer${index}`} size="m" />);
-  }
-  return acc;
-};
-
-const getDescriptions = (appDescriptions: string[]) =>
-  appDescriptions
-    .map(getDescriptionText)
-    .reduce<JSX.Element[]>(addSpacersBetweenElementsReducer, []);
-
 interface Props {
   addBasePath: (path: string) => string;
   solution: FeatureCatalogueSolution;
-  apps?: FeatureCatalogueEntry[];
 }
 
-export const SolutionPanel: FC<Props> = ({ addBasePath, solution, apps = [] }) => {
+export const SolutionPanel: FC<Props> = ({ addBasePath, solution }) => {
   const { trackUiMetric } = getServices();
 
+  const getSolutionGraphicURL = (solutionId: string) =>
+    `/plugins/kibanaReact/assets/solutions_${solutionId}_2x.png`;
+
   return (
     <EuiFlexItem
-      key={solution.id}
+      className="homSolutions__item"
       data-test-subj={`homSolutionPanel homSolutionPanel_${solution.id}`}
-      className={`${
-        solution.id === 'kibana' ? 'homSolutions__group homSolutions__group--single' : ''
-      } homSolutions__item`}
-      grow={1}
     >
-      <a
+      <EuiCard
         className={`homSolutionPanel homSolutionPanel--${solution.id}`}
+        description={solution.description}
         href={addBasePath(solution.path)}
+        icon={
+          <EuiToken
+            className="homSolutionPanel__icon"
+            iconType={solution.icon}
+            shape="circle"
+            size="l"
+          />
```

Comment:
😍 🎉 

Quick suggestion here to match the new Add Data pages, that was just merged into the page template. The solution logos (with some added texture) can be reused with `KibanaPageTemplateSolutionNavAvatar`. Then you also won't need the custom styles.

```suggestion
          <KibanaPageTemplateSolutionNavAvatar
            name={solution.title}
            iconType={solution.icon}
            size="xxl"
          />
```

Reply:
Made this change and also changed the solution PNG files to SVG, per your request. Thanks!